docker-compose-turtlebot.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+services:
+  isar:
+    build:
+      context: .
+      args:
+        ROBOT_REPOSITORY_CLONE_URL: https://${GITHUB_ACCESS_TOKEN}@github.com/equinor/isar-turtlebot.git
```

Comment:
Why do we need the github access token to clone isar-turtlebot?

Reply:
Ahh, good point. The docker-compose file was first made to be one file that could work for all robots, even those in a private repository. However, it then changed to be individual files for each as anything else didn't really work out. 

I will remove it.